samigo/samigo-app/src/webapp/jsf/author/item/fillInTheBlank.jsp

```diff
@@ -39,8 +39,19 @@
           $(function () {
               defining_answers = $("#defining_answers").html();
               mutually_exclusive = $("#mutually_exclusive").html();
-              checkMarkers();
-              $("#itemForm\\:markers").change(function () {
+              var markers = $("#itemForm\\:newmarkers").val();
```

Comment:
I'd use a const here. I've added eslint to the webcomponents project and banned vars there as they are brittle, so we may as well start using const where we can.